src/EnergyPlus/CurveManager.hh

```diff
@@ -84,7 +84,6 @@ namespace CurveManager {
     {
```

Comment:
That's an easy change, with lots of file diffs.  But no results diffs.  Kinda like the nature of this PR.  So I can bite the bullet and make that change.

Reply:
Changed the namespace to Curve.

Reply:
Not sure I agree with `Curve` as the new name esp since table lookups are in here as well, but "curve" is all over .  Are we planning to change the file names to match?

Reply:
No need to necessarily change the file name, but as we are moving towards eliminating `using` statements, we need to start consolidating namespaces and moving to shorter namespace names. The standard template library namespace is `std` not `StandardTemplateLibraryManager` for a reason (don't get me started on Nils Lohmann). 20-ish namespaces of 5 letters or less each should be enough.

Reply:
Hmmm, you mean you don't like `ZoneTempPredictorCorrector` as a namespace?

Reply:
At least it's not `ZoneTempPredictorCorrectorManager` and at least it's not called from a ton of other places like `Curve` is but yes ... it's not a good namespace name.

Reply:
80/20 rule, it's more important to do this for namespaces that are used in many other places, e.g., `DataGlobalConstants` (`GCnst`? `GlobC`?), `DataHVACGlobals` (`HCnst`? `HVACC`?), `UtilityRoutines` (`Util`? `Utils`?), `DataHeatBalance` (`HtBal`?), `FluidProperties` (`Fluid`?).

Reply:
I like moving in that direction.  Could we apply the 80/20 to the length of the names as well rather than firmly going down to 5 characters?  Some nice ones could be: 
 - DataGlobalConstants -> Constants
 - DataHVACGlobals -> HVAC (this namespace could be reused for all HVAC-related modules, not just data hvac globals)
 - UtilityRoutines -> Utility  (although I think it's probably better to just eliminate this dumping ground is probably better suited to be eliminated)
 - DataHeatBalance -> This module is full of a ton of different things, and I think should be split into the internal gains, zone/space specification, etc.
 - FluidProperties -> Fluid
 - Psychrometrics -> Psych
 - DataPlant -> Plant -- and everything specifically-plant-related should just be in the same Plant namespace.

As you mentioned 20-ish namespaces is a nice target for the different domains of EnergyPlus.  One complication is how to namespace things that touch multiple domains.  A hot water coil exists both on the HVAC side and the Plant side.  So what namespace does it live?  Group component models together?  Comps?  I'm not sure.  But I'm sure we could come up with something good.

Reply:
Ok, I'm fine with names that are a bit longer especially if it makes them pronounceable or easier to figure out what they are, but let's try to stay close to five if we can, e.g., `Consts` or `Convect` are fine.

Reply:
Yes, absolutely agree.  As close to five but with slightly longer instances being the exception.

Reply:
Nothing is a hard and fast rule obviously. It's programming, you can pretty much do whatever you want. 

The overarching point is that we are not really using namespaces/modules in the way that they were designed to be used. We started using them as a way to deconflict global variables from different files by making each file its own namespace (with the default being that the name of the namespace is the name of the file). We shouldn't need them for that anymore because we technically shouldn't be using global variables anymore (everything that isn't nailed down as a `constexpr` should be in `state`), but the namespaces have stuck.

Fine, they are still somewhat useful, but there is nothing that says they need to correspond one-to-one with files, much less have the same name as files. They should delineate definitions and code that belong together and be easy to use, and having separate namespaces for `ConvectionConstants` and `ConvectionCoefficients` with long names each accomplishes neither of those goals.